LogReader/LogReader/Models/Log.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+
+namespace LogReader.Models
```

Comment:
Path is LogReader/LogReader?

Reply:
> Path is LogReader/LogReader?

I added a folder where I created the project, that's why there's such path